client/src/controllers/scores/scores_controller.js

```diff
@@ -21,12 +21,19 @@ class ScoresController {
 		this.$scope.$on('reload', () => self.load(true))
 		this.Messanger.on('reload', () => self.load(false), true)
 		this.Tournament.teams().then(teams => {
-			this._loading = false
 			this._teamNumberList = []
 			for (var i = 0; i < teams.length; i++) {//Creates list of team numbers.
 				this._teamNumberList.push(teams[i].number)
 			}
 		})
+		this.Tournament.teams_rounds().then(rounds => {

```

Comment:
`javascript` conventions says that all names must be written in camelCase, so change `teams_rounds` to `teamsRounds`